pkg/profiles/profiles.go

```diff
@@ -89,11 +97,17 @@ func ToRoute(route *sp.RouteSpec) (*pb.Route, error) {
 		}
 		rcs = append(rcs, pbRc)
 	}
+	timeout, err := time.ParseDuration(route.Timeout)
```

Comment:
Can we also update the "no invalid service profiles" check to validate that the `Timeout` field contains a valid duration?